spec/spec_helper.rb

```diff
@@ -24,6 +24,8 @@
   config.infer_spec_type_from_file_location!
   config.use_transactional_fixtures = false
 
+  config.extend Spree::TestingSupport::AuthorizationHelpers::Controller, type: :system
```

Comment:
This should probably go into `solidus_dev_support` right?

Reply:
It could be a good idea, I'm going to ask if I can open this kind of PR there.